File: Bit Manipulation 2/BitManipulation2.java
public class BitManipulation2 {
    // boolean checkBit(int n, int i) {
    // return ((n >> i) & 1) == 1;
    // }

    boolean checkBit(int n, int i) {
        return (n & (1 << i)) != 0;
    }

    int setBits(int x, int y) {
        return (1 << x) | (1 << y);
    }

    int setIthBit(int n, int i) {
        return n | (1 << i);
    }

    public static void main(String[] args) {
        int ans = new BitManipulation2().setBits(3, 3);
        System.out.println(ans);
    }
}

File: Bit Manipulation 2/BitManipulation2.py
def checkBit(n, i):
    return ((n >> i) & 1) == 1


def checkBit(n, i):
    return (n & (1 << i)) != 0


def setBits(x, y):
    return (1 << x) | (1 << y)


def setIthBit(n, i):
    return n | (1 << i)


print(checkBit(82, 1))
print(checkBit(82, 2))
print(checkBit(82, 3))
